ci/build_docs.sh

```diff
@@ -35,6 +35,11 @@ rapids-mamba-retry install \
 
 export RAPIDS_DOCS_DIR="$(mktemp -d)"
 
+ANY_FAILURES=0
```

Comment:
This seems unused.
```suggestion
```

Reply:
Ack thanks for that. From an earlier version without `trap`.